manifests/1.3.7/opensearch-dashboards-1.3.7.yml

```diff
@@ -28,7 +28,7 @@ components:
     working_directory: gantt-chart
     ref: tags/1.3.7.0
   - name: observabilityDashboards
-    repository: https://github.com/opensearch-project/dashboards-observability.git
+    repository: https://github.com/opensearch-project/observability.git
```

Comment:
Since this is public? Should we be pointing the repo to https://github.com/opensearch-project/dashboards-observability

Reply:
If we change it to dashboards-observability, the manifest check would be failing as we haven't created tags for that repo. I think we would need some discussions for how to modify our manifests accordance to these repo name changes.

Reply:
@gaiksaya @zelinh +1 to what Zelin is saying I don't think it's right to use `dashboards-observability` in this case, or at least if it should be changed then all of the manifest files should be changed in my opinion. Please let me know of the result of those discussions, I can modify the PR as necessary or let me know if I can be of any help in those discussions. Thanks!